test/fixtures/output/kafka/consumer-group/describe.golden

```diff
@@ -0,0 +1,8 @@
++--------------------+------------------------------------------------------+
+| Cluster            | lkc-1234                                             |
+| Consumer Group     | consumer-group-1                                     |
+| Coordinator        | broker-1                                             |
+| Simple             | true                                                 |
+| Partition Assignor | org.apache.kafka.clients.consumer.RoundRobinAssignor |
```

Comment:
Assigner?

Reply:
They're both words (with similar but different meanings), and assignor is the one that's used in the API.